features/messages/impl/src/test/kotlin/io/element/android/features/messages/mediaplayer/MediaPlayerFake.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.messages.mediaplayer
+
+import io.element.android.features.messages.impl.mediaplayer.MediaPlayer
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+
+/**
+ * Fake implementation of [MediaPlayer] for testing purposes.
+ */
+class MediaPlayerFake : MediaPlayer {
```

Comment:
I think the naming convention we're using is `FakeClassName` e.g. `FakeMediaPlayer`

Reply:
Changed!